api/routes/application/routes.py

```diff
@@ -184,9 +185,16 @@ def submit(self, application_id):
             # assessment service to import the application
             #  (currently assessment is using a CRON timer to pick up messages,
             # not a webhook for triggers)
-            submit_message_to_queue(
-                application_with_form_json,
-                application_attributes,
+
+            # TODO: (FS-3703) Revisit this part after AWS migration
+            # 'MessageGroupId' & 'MessageDeduplicationId' are mandatary parameters to be provided on PAAS,
+            # while they are not acceptable parameters on localstack queue
+            _SQS_CLIENT.submit_single_message(
+                queue_url=Config.AWS_SQS_IMPORT_APP_PRIMARY_QUEUE_URL,
+                message=application_with_form_json,
+                extra_attributes=application_attributes,
+                message_group_id="import_applications_group",
```

Comment:
This group id could maybe be made a constant?

Reply:
the `group_id` is mandatory parameter to be provided on PAAS. Need to rework on args for submit message function after the AWS migration. There's a `TODO` comment with JIRA ticket added on the above lines